README.md

```diff
@@ -71,6 +71,14 @@ See [related documentation](./guides/alerting/alerting.md) for more information.
 
 **Trento** can be installed in various ways, depending on your needs and the requirements of its different components. Please refer to the specific [Agent](https://github.com/trento-project/agent) documentation for more information.
 
+## Installing Trento manually
```

Comment:
I don't like the `manually` term.
What does it mean?
I would go simply with `Installing Trento`.
The same for the body of the chapter. `To install Trento, please refer to the